python/pyspark/worker.py

```diff
@@ -140,15 +139,20 @@ def read_single_udf(pickleSer, infile, eval_type):
         else:
             row_func = chain(row_func, f)
 
+    # make sure StopIteration's raised in the user code are not
+    # ignored, but re-raised as RuntimeError's
+    func = fail_on_stopiteration(row_func)
```

Comment:
I think `row_func` name was fine. Let's just leave it as was.

Reply:
I wanted to avoid the overhead of calling get_argspec even when it's not needed

Reply:
Ah, sure.